inferential/static/playground.js

```diff
@@ -59,3 +59,20 @@ async function displayCompletion() {
   button.disabled = false
   loadingSpinner.classList.add('spinner-hidden')
 }
+
+//Displays text from prompts based on which model is selected
+const promptLong = document.getElementById('promptLong')
+const promptDesc = document.getElementById('PromptDesc')
```

Comment:
I would ask tomorrow, but I think we should move these variables to the top of the file where the rest of them are.